content/get-started.md

```diff
@@ -2,6 +2,18 @@
 title: Get Started with Webpack
 ---
 
-* basic concepts, get webpack up and running, technical focus
-* target - the user should be able to complete this in 10 minutes
+`webpack` is a module bundler.A bundler is a tool that builds all your project files to a single asset file that can be loaded onto a browser.It requires that your application files be written as [modules](/concepts/modules).
+The project structure consists of a single entry file with references to other modules as dependencies creating a tree like dependency graph.The dependency modules can belong to your application or can be third party libraries.
+
+The bundler creates this dependency graph and parses through it to include all the required modules in the final bundle.The final bundle file will contain all the modules in the right order of dependency and none of the modules will be duplicated.
+Other examples of module bundlers are [browserify]() and [rollupjs]()
+
+Unlike module bundlers, [Grunt](http://grunt),[Gulp](),[Broccoli]() are task runners.
```

Comment:
Yes, I'm also pretty disappointed with the description I came up with. It all sounds very theorectical as you said. I only wanted to get across the idea of a module bundler and that webpack is a great one at that. Would you help me draft this again? Can I DM you?